File: src/main/java/cz/muni/fi/pa039/widgets/TextWidget.java
package cz.muni.fi.pa039.widgets;

import cz.muni.fi.pa039.Point;

public class TextWidget extends AbstractWidget {

    public Point bottomLeft;
    public String text;

    private int id;

    public TextWidget(Point bottomLeft, String text, int id) {
        this.bottomLeft = bottomLeft;
        this.text = text;
        this.id = id;
    }

    @Override
    public int getId() {
        return id;
    }

    @Override
    public int getFill() {
        return 0;
    }

    @Override
    public boolean pointIsInside(int x, int y) {
        return false;
    }
}

File: src/main/java/cz/muni/fi/pa039/widgets/TextWidget.kt
package cz.muni.fi.pa039.widgets

import cz.muni.fi.pa039.Point

class TextWidget(var bottomLeft: Point, var text: String, override val id: Int) : AbstractWidget {
    override var isSelected = false
    override val fill = 0
    override fun pointIsInside(x: Int, y: Int): Boolean {
        return false
    }
}
